mypy.ini

```diff
@@ -0,0 +1,6 @@
+[mypy]
+python_version = 3.7
```

Comment:
I haven't used mypy before. Is it typical to use 3.7 for type checking even when the version you are running is 3.9?

Reply:
No, you this was just from experiments to check multiple different python version.
Will change to 3.9